scripts/bootstrap.sh.in

```diff
@@ -182,7 +190,8 @@ main() {
     run "Installing mandatory packages" install_packages "${PACKAGES[@]}"
     run "Configuring Salt minion to run in local mode" configure_salt_minion_local_mode
     run "Ensure archive is available" ensure_archives_mounted
-    run "Checking local node" check_local_node
+
+    check_bootstrap_node
```

Comment:
I would rather use the `run` function as it is done above, it rationalizes the output from this script.
```suggestion
    run "Checking bootstrap node" check_bootstrap_node
```

Reply:
To me, we do not need a run for the CA minion check. Since it's just a check that basically do not output anything

Reply:
Except if one of the command fail or if it fails itself.

Reply:
Sure, but in this case, we will exit anyway. The only impact is that we will not log the error in the log file but only in stdout. Anyway ok I will update it. It does not cost much :smile:

Reply:
Ok I just split CA minion check and "local node" checks